LICENCE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
@zackads As discussed, let's remove the license for now as we do not want to be making decision on behalf of the MCA.  Aniko is aware and will be able to provide us with an answer on what license should be used and wording around copyright.